amplitude.js

```diff
@@ -229,6 +237,14 @@ Amplitude.prototype.init = function(apiKey, opt_userId, opt_config) {
           //log(e);
         }
       }
+      var savedUnsentIdentifysString = localStorage.getItem(this.options.unsentIdentifyKey);
+      if (savedUnsentIdentifysString) {
+        try {
+          this._unsentIdentifys = JSON.parse(savedUnsentIdentifysString);
+        } catch (e) {
+          //log(e);
+        }
+      }
```

Comment:
It's not much code, but I'd consider refactoring this so that it can be reused for events/identifies.